### PR TITLE
Menu order fix for IE

### PR DESCRIPTION
**Pull request by wmariz**
Add a default name if "menu name" setting is blank (base.html)

Change "item" variable name to "menu_item" since "item" is a reserved
word (default.js)

### CODE REVIEW

**Comment by wmariz** [comment]
I think there is a difference between using "item" as a function parameter and inside a function (like I did in this for loop).

I just capitalized for esthetics, but I think is better to use "m.get('name')" instead of "m.get('id').capitalize()"

I will post a new commit.

**Comment by Hellowlol** [comment]
Good, now ppl make there own name. Like tv instead of sonarr etc

**Comment by wmariz** [comment]
Sorry, I didn't understand.

**Comment by Hellowlol** [comment]
Sorry. In the settings page  there is field called module_name (sickrage_name etc), ppl can use that field to display a other name then sickrage.

This field should be default in the navbar and it it isnt set it should show the id.

File: interfaces/default/html/base.html [hunk 1]
```diff
@@ -162,7 +162,7 @@ <h3 class="modal-h3"></h3>
     menus = {}
     % for m in htpc.MODULES:
         % if s.get(m.get('id')+'_enable') and m.get('id') != 'nzbsearch':
-            menus['nav-${m.get('id')}'] = '<li class="nav-menu-item" id="nav-${m.get('id')|h, entity}"><a href="${htpc.WEBDIR}${m.get('id')|h, entity}/">${s.get(m.get('id')+'_name')|h, entity}</a></li>';
+                menus['nav-${m.get('id')}'] = '<li class="nav-menu-item" id="nav-${m.get('id')|h, entity}"><a href="${htpc.WEBDIR}${m.get('id')|h, entity}/">${m.get('name') if s.get(m.get('id')+'_name').strip() == "" else s.get(m.get('id')+'_name')|h, entity}</a></li>';
```
Comment (Hellowlol) [inline interfaces/default/html/base.html]: ${s.get(m.get('id')+'_name') else m.get('name'))|h, entity}

**Comment by Hellowlol** [comment]
Ill test this and merge after i get home from work.

**Comment by wmariz** [comment]
I think it is working the way you want, because python inline if statements have the following sintax:

"true_value" if "condition" else "false_value"

In other words, if "custom name" is blank, it will use the "default name" otherwise it will use the "custom name".

**Comment by Hellowlol** [comment]
Yeah I noticed that now.

**Pull request merged**